doc/getting_started/index.ipynb

```diff
@@ -484,6 +499,18 @@
     "df_penguins.hvplot.scatter(x='bill_length_mm', y='bill_depth_mm', by='sex')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "77b34a64-a3f8-468a-b263-0b186bff6f0b",
+   "metadata": {},
+   "source": [
+    "## Rendering and saving output\n",
+    "\n",
+    "hvPlot needs to run in a Jupyter environment to automatically show output plots. However, if you are using a raw Python or IPython console, it is still possible to show the plots with `hvplot.show()` or save the plots with `hvplot.save(plot, 'this_plot.html')` even if you cannot view them interactively.\n",
```

Comment:
Suggesting another wording (please improve the text as you see fit if you like it) as I felt it sounded like hvPlot needed to be run in Jupyter.

> Objects returned by hvPlot calls are automatically displayed in a notebook environment (e.g. Jupyter, VS Code), like on this page that is built from a Jupyter notebook. If you are running your code outside of a notebook, you can view plots with `hvplot.show(<obj>)` and save plots with `hvplot.save(<obj>, 'this_plot.html')`. For more information about viewing or saving plots (e.g. as PNG), check out the user guide section on [viewing plots](https://hvplot.holoviz.org/user_guide/Viewing.html).
  Tips: Bokeh plots include by default a "Save" icon in their toolbar.